apps/src/blockly/addons/cdoXml.js

```diff
@@ -46,16 +47,37 @@ export default function initializeBlocklyXml(blocklyWrapper) {
    * @returns {Object[]} An array of objects containing the created blocks and their positions.
    */
   blocklyWrapper.Xml.domToBlockSpace = function (workspace, xml) {
-    const partitionedBlockElements = getPartitionedBlockElements(
+    const {prioritizedBlocks, remainingBlocks} = getSplitBlockElements(
       xml,
       PROCEDURE_DEFINITION_TYPES
     );
     const blocks = [];
+
+    for (const xmlFunctionBlock of prioritizedBlocks) {
```

Comment:
Would highlighting that prioritized blocks == definition blocks by aliasing when we destructure add clarity?

```
const { prioritizedBlocks: definitionBlocks, remainingBlocks: callBlocks } = getSplitBlockElements(XML, PROCEDURE_DEFINITION_TYPES);
```

I don't have a strong opinion either way, but I did read this and have to remind myself why just iterating through the `prioritizedBlocks` was sufficient (because they're the only definition blocks). Your comment addresses it too though.